docs/tutorials/python/ndarray.md

```diff
@@ -94,9 +96,9 @@ If two `NDArray`s are located on different devices, we need to explicitly move t
 
 ## Load and Save
 
-There are two ways to save data to (or load it from) disks easily. The first way uses
-`pickle`.  `NDArray` is pickle compatible, which means that you can simply pickle the
-`NDArray` as you do with `numpy.ndarray`:
+There are two ways to save data to (or load it from) disks easily. The first way
+uses `pickle`.  A `NDArray` is pickle compatible, which means that you can
```

Comment:
I believe we don't need an indefinite article here because we mean to say that generally the class is pickle-compatible (not just a single instance). Also pickle compatible is a two-word adjective, so we should hyphenate.